docs/publishing/sections/8_ckan.md

```diff
@@ -77,22 +86,22 @@ An example from the latest-only GTFS data exposure.
                 https://gtfs.org/reference/static#attributionstxt.
 ```
 
-### Publish the data!
+### Publish the data
+
 If you are using dbt-based publishing, the `publish_exposure` subcommand of `publish.py`
 will query BigQuery, write out CSV files, and upload those files to CKAN.
-Either create an Airflow job to refresh/update the data at the specified
-frequency, or do it manually. You can use flags to execute a dry run or write to
-GCS without also uploading to CKAN.
+[An Airflow job](https://o1d2fa0877cf3fb10p-tp.appspot.com/dags/publish_open_data/grid) refreshes/updates the data at a specified
+frequency, or the publication script can be run manually. By default, the `--no-publish` flag is set, executing a dry run. You can also write to GCS without uploading to CKAN by manually entering an arbitrary bucket destination.
 
-The publish script supports referencing `gs://` paths for the manifest; by default,
-the script will read the latest manifest in GCS uploaded by an Airflow job.
+The publish script supports referencing `gs://` paths for the manifest, which is used to determine which tables and columns to publish; by default, the script will read the latest manifest in GCS uploaded by the `dbt_run_and_upload_artifacts` Airflow job.
```

Comment:
weekly publishing job

Reply:
Resolved in [4d6b182](https://github.com/cal-itp/data-infra/pull/2839/commits/4d6b182483b5a630658e3f6f2d2ed0d2f5ea1647)